ur_description/config/ur5e/visual_parameters.yaml

```diff
@@ -1,67 +1,67 @@
 mesh_files:
   base:
     visual:
-      mesh: "package://ur_description/meshes/ur5e/visual/base.dae"
+      mesh: "ur5e/visual/base.dae"
       material:
         name: "LightGrey"
         color: "0.7 0.7 0.7 1.0"
     collision:
-      mesh: "package://ur_description/meshes/ur5e/collision/base.stl"
+      mesh: "ur5e/collision/base.stl"
 
   shoulder:
     visual:
-      mesh: "package://ur_description/meshes/ur5e/visual/shoulder.dae"
+      mesh: "ur5e/visual/shoulder.dae"
       material:
         name: "LightGrey"
         color: "0.7 0.7 0.7 1.0"
     collision:
-      mesh: "package://ur_description/meshes/ur5e/collision/shoulder.stl"
+      mesh: "ur5e/collision/shoulder.stl"
 
   upper_arm:
     visual:
-      mesh: "package://ur_description/meshes/ur5e/visual/upperarm.dae"
+      mesh: "ur5e/visual/upperarm.dae"
       material:
         name: "LightGrey"
         color: "0.7 0.7 0.7 1.0"
     collision:
-      mesh: "package://ur_description/meshes/ur5e/collision/upperarm.stl"
+      mesh: "ur5e/collision/upperarm.stl"
       mesh_files:
```

Comment:
What is the line for? Its only listed for this link.

Reply:
Good question...